internal/nodes/avi_model_l4_translator.go

```diff
@@ -294,6 +296,46 @@ func PopulateServersForNPL(poolNode *AviPoolNode, ns string, serviceName string,
 	v6ServerCount := 0
 	var poolMeta []AviPoolMetaServer
 
+	// create a mapping from pod name to its endpoint condition
+	conditionMap := map[string]*bool{}
+	if lib.AKOControlConfig().GetEndpointSlicesEnabled() {
+		epSliceIntList, err := utils.GetInformers().EpSlicesInformer.Informer().GetIndexer().ByIndex(discovery.LabelServiceName, ns+"/"+serviceName)
+		if err != nil {
+			utils.AviLog.Warnf("key: %s, msg: error while retrieving endpointsice: %s", key, err)
+			return nil
+		}
+		for _, epSliceInt := range epSliceIntList {
+			epSlice, isEpSliceClass := epSliceInt.(*discovery.EndpointSlice)
+			if !isEpSliceClass {
+				// not an epslice. continue
+				utils.AviLog.Warnf("key: %s, msg: invalid endpointslice object", key)
+				continue
+			}
+			// select epslice containing target port
+			var found = false
```

Comment:
can we replace it with `found := false`?

Reply:
Done